.github/actions/build-name/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "build names"
+outputs:
+  short_sha:
+    value: ${{ steps.sha.outputs.SHORT_SHA }}
+  pretty_branch_name:
+    value: ${{ steps.branch.outputs.NAME }}
+  artifact_prefix:
+    value: ${{ steps.branch.outputs.NAME }}-${{ steps.sha.outputs.SHORT_SHA }}
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v3
+    - name: Set short sha output
+      id: sha
+      run: echo "SHORT_SHA=$(git rev-parse --short=11 HEAD)" >> $GITHUB_OUTPUT
```

Comment:
Hi @NickAzureDevops, @JoshuaLicense has rewritten this action to get the pretty branch name. I suggest you remove this actions from this PR and use the one Josh will be adding shortly instead.

Reply:
@AnnaDodson  Done. Removed it from the root of the readme.md so @JoshuaLicense  will have to add it again once he added his action